.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         include:
           - # If upgrading the Haskell image, also upgrade it in the lint job below
             os: "ubuntu-latest"
-            image: "haskell:8.10.7-stretch@sha256:100f8fb7d7d8d64adb5e106fe8136b8d4cbdc03aeb2cbd145a7597d74b69bafb"
+            image: "haskell:9.2.2-buster@sha256:c2f489a95be8dd0b253da6980420a6e9c02e81c2d14838ca774f86f5b5db91f4"
```

Comment:
`buster` (Debian 10) is the next Debian release after `stretch`. There is no `stretch` image which includes GHC 9.2.2, probably because `stretch` will stop receiving security updates next month (June 2022).

Reply:
Looks like this would mean bumping our glibc dependency from 2.24 to 2.28.

Should we make a Discourse post asking if people would object to that? Seems like we consistently stir up trouble whenever this changes.

Reply:
Yes, good point. This certainly isn’t ideal. Maybe we should just take a basic Debian stretch image and install the Haskell tools ourselves? My inclination is to avoid bumping this if we can manage it.